pkg/helm/helm_test.go

```diff
@@ -127,6 +174,54 @@ func TestRegistryLoginCommand(t *testing.T) {
 				"pass",
 			},
 		},
+		{
+			name: "Valid registry with auth and skip TLS verify",
+			host: "registry-1.docker.io",
+			repo: &image.HelmRepository{
+				Name: "apache-repo",
+				URL:  "oci://registry-1.docker.io/bitnamicharts/apache",
+				Authentication: image.HelmAuthentication{
+					Username: "user",
+					Password: "pass",
+				},
+				SkipTLSVerify: true,
+			},
+			expectedArgs: []string{
+				"helm",
+				"registry",
+				"login",
+				"registry-1.docker.io",
+				"--username",
+				"user",
+				"--password",
+				"pass",
+				"--insecure",
+			},
+		},
+		{
+			name: "Valid registry with auth and plain HTTP",
+			host: "registry-1.docker.io",
+			repo: &image.HelmRepository{
+				Name: "apache-repo",
+				URL:  "oci://registry-1.docker.io/bitnamicharts/apache",
+				Authentication: image.HelmAuthentication{
+					Username: "user",
+					Password: "pass",
+				},
+				SkipTLSVerify: true,
```

Comment:
This is duplicate of the previous test case.

Reply:
Good catch, meant for that to be `plainHTTP: true`.